Lib/test/test_typing.py

```diff
@@ -855,6 +855,15 @@ def test_accepts_single_type(self):
         (*tuple[int],)
         Unpack[Tuple[int]]
 
+    def test_dir(self):
+        dir_items = set(dir(Unpack[Tuple[int]]))
+        for required_item in [
+            '__args__', '__parameters__', '__origin__',
+            'copy_with',
```

Comment:
I'm not sure we should be asserting that `copy_with` is included in `dir()`, since it's an undocumented method that isn't public API. It feels like we'd be asserting undesirable behaviour. The fact that it is included in `dir()` currently could possibly be considered a bug.